admin-dev/themes/new-theme/scss/theme.scss

```diff
@@ -57,6 +57,7 @@
 @import "pages/employee";
 @import "pages/error";
 @import "pages/orders";
+@import "pages/login";
```

Comment:
login page includes this `admin-dev/themes/new-theme/scss/theme.scss` because it needs all the other theme assets. theme.scss then includes login page assets because they are needed in the login page and the styles don't seem to work if I change something with these imports :thinking:

Reply:
can be done as an improvement later 😄